check.sh

```diff
@@ -79,8 +67,7 @@ Problems were found by static analysis (listed above).
 To fix formatting and see remaining errors, run
 
     uv pip install -r test-requirements.txt
-    black src/trio
-    ruff check src/trio
+    pre-commit run --all-files
```

Comment:
hook stage manual?

Reply:
?
We don't have any manual hooks anymore

Reply:
ah I see, it might be worth using hook stage manual here, just in case any get added

Reply:
it feels kinda weird suggesting people do stuff that is completely redundant/ineffective. And if we do add manual stages there's perhaps gonna be a decent reason we make them manual? Idk I'll leave it as is /shrug